sdk/src/main/java/com/hedera/hashgraph/sdk/Network.java

```diff
@@ -36,13 +41,114 @@
 
         try {
             setNetwork(network);
-        } catch (InterruptedException e) {
-            // This should never occur. The network is empty.
-        } catch (TimeoutException e) {
+        } catch (InterruptedException | TimeoutException e) {
             // This should never occur. The network is empty.
         }
     }
 
+    static Network forNetwork(ExecutorService executor, Map<String, AccountId> network) {
+        return new Network(executor, network);
+    }
+
+    static Network forMainnet(ExecutorService executor) {
```

Comment:
Add `@SuppressWarnings("java:S1313")` to this and other methods with IPs to suppress the numerous sonar warnings.